packages/fluentui/react-northstar/src/themes/teams-dark/components/Menu/menuVariables.ts

```diff
@@ -53,4 +53,5 @@ export const menuVariables = (siteVars: any): Partial<MenuVariables> => ({
   activePrimaryIndicatorColor: siteVars.colors.white,
   activePrimaryVerticalIndicatorColor: siteVars.colors.white,
   indicatorColorHover: siteVars.colors.white,
+  subMenuIconColor: undefined,
```

Comment:
I don't think this should be here...? if you set the token in the default theme, the color will come through. And in your styles you're only showing the filled state if this is set so in dark & high contrast themes you're not seeing the right design.
![image](https://user-images.githubusercontent.com/828692/193655776-7ad88cee-85d0-4e28-9ced-a8fcf26c49e5.png)

![image](https://user-images.githubusercontent.com/828692/193655802-78a0fb89-944e-4930-804d-cf5408baa7ef.png)

Reply:
You are totally right, I took as a reference the wrong theme in v9. I update it immediately, thank you!